src/sentry/options/defaults.py

```diff
@@ -10,15 +10,38 @@
 from sentry.options import (
     FLAG_IMMUTABLE, FLAG_NOSTORE, FLAG_PRIORITIZE_DISK, FLAG_REQUIRED, register
 )
-from sentry.utils.types import Dict, String
+from sentry.utils.types import Dict
 
+
+def default_from_email():
```

Comment:
@dcramer Are you adverse to this change? It's sorta unrelated to this PR, but the default `root@localhost` bothered me. :)

In Gmail at least, Google seems to mark emails with "[Sentry] " in subject coming from root@localhost as spam. I get bit by this testing, so our out of the box experience is that emails get sent spam.

Obviously, we can/should encourage changing this default value. I'm likely going to move it into the welcome/setup screen, but I think we should do a more sensible default than `root@localhost`.